### ISSUE TITLE
Mention policies you announced somewhere more accessible

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-www#6987]
I personally think that the policies like your [Browser Extension/Userscript Policy](https://scratch.mit.edu/discuss/topic/284272/) or [Cloud chat projects](https://scratch.mit.edu/discuss/topic/357609/) announcements in the Discussion forums should be easier to find, like maybe you could list them in your Community Guidelines. I never stop seeing people mention browser extensions on the Discussion forums by accident, so I think saying something about browser extensions on the Community Guidelines would be beneficial.

**Comment by PPPDUD** [comment]
+1. This would clean up the Scratch community and would help with some very big problems.

Although may I please ask if you could also suggest for forum rules to be shown to first-time users on that forum? I'm 90% sure that this extra suggestion is a duplicate, but I have no link to prove it.

Offtopic: I am signing off soon because my sandwich is going cold slowly. Any responses will most likely be from tomorrow. Also, Grammarly needs to return to school. No offense to them by that! :)

**Comment by benjiwheeler** [comment]
I am labeling this with a priority, not because we're sure we want to do this, but because it's worth considering.

**Comment by CST1229** [comment]
> I am labeling this with a priority, not because we're sure we want to do this, but because it's worth considering.

Who in the world would

**Comment by PPPDUD** [comment]
Good point.

**Comment by derter-menter** [comment]
> I am labeling this with a priority, not because we're sure we want to do this, but because it's worth considering.

No offence, but who in the world would look in *forum topics* ( the hidden rotting place 90% of scratchers don’t know about) to find rules? Why aren't they elsewhere?

**Comment by Scratchfangs** [comment]
I agree, 2 years ago I had 0 idea that the policy was a thing. And I got a warn on my record for it.

**Comment by CocoTheMii** [comment]
> I am labeling this with a priority, not because we're sure we want to do this, but because it's worth considering.

With all due respect, why _wouldn't_ you want to do this? Does the Scratch Team benefit from keeping their guidelines intentionally vague?

Take face reveals, for example. Until around early 2022 (iirc), users were allowed to show their face in a project, provided that it was not the primary focus. However, this rule was suddenly changed to ban faces in projects altogether, **a change which was never formally announced.** As a result, many users remain unaware of the change, even today.

Transparency is key. It is not enough to simply _imply_ the existence of these rules (i.e. removing the camera button from the editor); it should be perfectly clear to any given user what is and is not allowed on Scratch. This means two things:

1. Announcing rule changes immediately upon implementation **at the very latest**, and in a location where it will be seen by as many Scratchers as possible
2. Providing an up-to-date list of every single rule not already defined in the Community Guidelines or Terms of Use

Otherwise, you simply cannot expect every user to follow every single rule that they don't know about.
